README.md

```diff
@@ -67,6 +67,8 @@ The meaning of each error code is given here:
 * `E1024` - Your [Discord guild](https://discord.com/developers/docs/resources/guild) does not contain a [role](https://discord.com/developers/docs/topics/permissions#role-object) with the name "**@Archivist**".
 (This [role](https://discord.com/developers/docs/topics/permissions#role-object) is required for the `/archive` [command](https://discord.com/developers/docs/interactions/application-commands))
 
+* `E1025` - Your [Discord guild](https://discord.com/developers/docs/resources/guild) does not contain a [role](https://discord.com/developers/docs/topics/permissions#role-object) with the name "@**Applicant**". (This [role](https://discord.com/developers/docs/topics/permissions#role-object) is required for the `/give-applicant-role` [command](https://discord.com/developers/docs/interactions/application-commands) and respective user and message commands)
```

Comment:
`@` symbol should also be capitalised (within `**` symbols): `... with the name "**@Applicant**". (This ...`